executor/runtime/docker/docker_linux.go

```diff
@@ -245,7 +270,11 @@ func runServiceInitCommand(ctx context.Context, log *logrus.Entry, cID string, r
 }
 
 func startSystemdUnit(ctx context.Context, conn *dbus.Conn, taskID string, cID string, runtime string, opts serviceOpts) error {
-	qualifiedUnitName := fmt.Sprintf("%s@%s.service", opts.unitName, taskID)
+	postFix := "service"
+	if opts.target {
```

Comment:
Why can't we just put `.service` or `.target` in the unit name itself?

Reply:
and parse / unparse it here? That seems uglier than just having a flag for it.

Reply:
No, just not even parse it at all. Why do we need to parse it?
Just print the whole unit name

Reply:
because you need to have something that inserts the container ID between the "@" and the "."